.idea/InnerEye-DeepLearning.iml

```diff
@@ -4,7 +4,7 @@
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
     </content>
-    <orderEntry type="jdk" jdkName="3.7 @ Ubuntu 20.04" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="3.7 @ Ubuntu-20.04" jdkType="Python SDK" />
```

Comment:
what if you manually set back to "3.7 @ Ubuntu 20.04"?

Reply:
Manually set back to "3.7 @ Ubuntu 20.04".